packages/storybook/stories/INTERNAL/UnsupportedBrowser/UnsupportedBrowser.stories.tsx

```diff
@@ -34,16 +36,20 @@ const getDocs: () => JSX.Element = () => {
     <Stack>
       <SingleLineBetaBanner />
       <Title>Unsupported Browser</Title>
-      <Description>Component to display help information when the app is loaded in a unsuppported browser.</Description>
+      <Description>
+        Component to display help information when the app is loaded in a unsuppported environment. Shows different
+        screens based on what the environment info from the calling SDK is reporting.
```

Comment:
nit: change to "based on the environment info the calling SDK is reporting"
possibl change calling SDK to "Azure calling communication service" to keep consistent with your comments in the code.